### DISCUSSION TITLE
Creating a conditional checkbox workflow [Jan 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4938] [Discord Threads]
> 🙋‍♂️ **Hirslay**

**Comment by o1lab** [comment]
> 🙋‍♂️ **kyleecodes**  
#### Tagging <@&682859083799199814> because I am curious about this too. Team should be able to confirm.  
#  
> 💁 **o1lab**  
#### This currently not possible - can envision this as part of automation feature in the future  
#  
> 🙋‍♂️ **kyleecodes**  
#### We can make this a feature request: https://github.com/nocodb/nocodb/issues/new?assignees=&labels=&template=feature_request.md&title=%5BFeature%5D+  
#  
> 💁 **o1lab**  
#### We shud have a feature request for automation  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1065297828659875982)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
